pf4j/src/main/java/ro/fortsoft/pf4j/DefaultPluginManager.java

```diff
@@ -131,6 +131,23 @@ public DefaultPluginManager(File pluginsDirectory) {
         initialize();
     }
 
+    /**
+     * Returns the currently used direcotry containing the plugins
+     * @return File pointing to the plugins directory
+     */
+    public File getPluginsDirectory() {
```

Comment:
In your scenario if I "relaxed" `pluginsDirectory` variable from "private" to "protected" I think that we resolve the problem.